### DISCUSSION TITLE
Get User Info for Target?

### DISCUSSION THREAD

**Discussion by joncashk** [Sehelitar/Kick.bot#61] [Q&A] [answered]
I'm trying to mirror my Streamer.bot Twitch alerts for Kick as well.  A lot of my alerts use the targetUserProfileImageUrl from the User Info for Target that is available by default, however I noticed that it isn't something that is available from this Kick integration.  Is this something that is possible/available from Kick?

**Comment by Sehelitar** [comment]
Hi! 👋 
This argument will be set only is that user has a profile picture set. Kick doesn't send a "default" profile picture back like Twitch does.
Although, in the next update, I'll try to fall back to the same image used by their web interface when the user doesn't have one set.

**Reply by joncashk** [reply]
Thank you so much!!!

**Comment by Sehelitar** [comment, marked as answer]
[A new update](https://github.com/Sehelitar/Kick.bot/releases/latest) (v0.4.1) is available and provide a link to the official default profile picture if the user didn't set one.
